Prysm will not compile for Windows

# 🐞 Bug Report

### Description
Current master will not compile for Windows on Windows using `go build` or if attempted to compile for Windows on Linux (Cross compile)

Output if compilation attempted on Windows using` go build .` inside validator folder:
`C:\Users\dv8si\go\pkg\mod\github.com\coreos\go-systemd@v0.0.0-20181012123002-c6f51f82210d\journal\journal.go:99:13: undefined: syscall.UnixRights`

Likely related to PR https://github.com/prysmaticlabs/prysm/pull/7429

Build attempt in Linux was by cd into validator folder and running:
`env GOOS=windows GOARCH=amd64 go build --tags=blst_enabled .`

Giving (in part):
```
# github.com/coreos/go-systemd/journal
/root/go/pkg/mod/github.com/coreos/go-systemd@v0.0.0-20181012123002-c6f51f82210d/journal/journal.go:99:13: undefined: syscall.UnixRights
```

## 🔬 Minimal Reproduction

Try to build current master for Windows.


## 🌍  Your Environment
Windows 10 AMD64.  Also tried compiling within WSL2 (Ubuntu) for Windows as well as in Docker within Ubuntu VM within Windows.  I have not tried on native Linux.

--- a/beacon-chain/BUILD.bazel
+++ b/beacon-chain/BUILD.bazel
@@ -28,7 +28,6 @@ go_library(
         "@com_github_sirupsen_logrus//:go_default_library",
         "@com_github_urfave_cli_v2//:go_default_library",
         "@com_github_urfave_cli_v2//altsrc:go_default_library",
-        "@com_github_wercker_journalhook//:go_default_library",
         "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
     ],
 )
@@ -58,7 +57,6 @@ go_image(
         "//shared/debug:go_default_library",
         "//shared/featureconfig:go_default_library",
         "//shared/logutil:go_default_library",
-        "//shared/maxprocs:go_default_library",
         "//shared/version:go_default_library",
         "@com_github_ethereum_go_ethereum//log:go_default_library",
         "@com_github_ipfs_go_log_v2//:go_default_library",
@@ -66,8 +64,8 @@ go_image(
         "@com_github_sirupsen_logrus//:go_default_library",
         "@com_github_urfave_cli_v2//:go_default_library",
         "@com_github_urfave_cli_v2//altsrc:go_default_library",
-        "@com_github_wercker_journalhook//:go_default_library",
         "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
+        "//shared/maxprocs:go_default_library",
     ],
 )
 

--- a/beacon-chain/main.go
+++ b/beacon-chain/main.go
@@ -137,8 +137,6 @@ func main() {
 			logrus.SetFormatter(f)
 		case "json":
 			logrus.SetFormatter(&logrus.JSONFormatter{})
-		case "journald":
-			journalhook.Enable()
 		default:
 			return fmt.Errorf("unknown log format %s", format)
 		}

--- a/beacon-chain/main.go
+++ b/beacon-chain/main.go
@@ -21,7 +21,6 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
 	"github.com/urfave/cli/v2/altsrc"
-	"github.com/wercker/journalhook"
 	prefixed "github.com/x-cray/logrus-prefixed-formatter"
 )
 

--- a/deps.bzl
+++ b/deps.bzl
@@ -3630,9 +3630,3 @@ def prysm_deps():
         sum = "h1:tYH07UPoQt0OCQdgWWMgYHy3/a9bcxNpBIysykNIP7I=",
         version = "v2.20.5+incompatible",
     )
-    go_repository(
-        name = "com_github_wercker_journalhook",
-        importpath = "github.com/wercker/journalhook",
-        sum = "h1:shC1HB1UogxN5Ech3Yqaaxj1X/P656PPCB4RbojIJqc=",
-        version = "v0.0.0-20180428041537-5d0a5ae867b3",
-    )

--- a/go.mod
+++ b/go.mod
@@ -106,7 +106,6 @@ require (
 	github.com/wealdtech/go-eth2-wallet-nd/v2 v2.3.0
 	github.com/wealdtech/go-eth2-wallet-store-filesystem v1.16.1
 	github.com/wealdtech/go-eth2-wallet-types/v2 v2.6.0
-	github.com/wercker/journalhook v0.0.0-20180428041537-5d0a5ae867b3
 	github.com/x-cray/logrus-prefixed-formatter v0.5.2
 	go.etcd.io/bbolt v1.3.4
 	go.opencensus.io v0.22.4

--- a/shared/cmd/flags.go
+++ b/shared/cmd/flags.go
@@ -167,7 +167,7 @@ var (
 	// LogFormat specifies the log output format.
 	LogFormat = &cli.StringFlag{
 		Name:  "log-format",
-		Usage: "Specify log formatting. Supports: text, json, fluentd, journald.",
+		Usage: "Specify log formatting. Supports: text, json, fluentd.",
 		Value: "text",
 	}
 	// MaxGoroutines specifies the maximum amount of goroutines tolerated, before a status check fails.

--- a/slasher/BUILD.bazel
+++ b/slasher/BUILD.bazel
@@ -24,7 +24,6 @@ go_library(
         "@com_github_sirupsen_logrus//:go_default_library",
         "@com_github_urfave_cli_v2//:go_default_library",
         "@com_github_urfave_cli_v2//altsrc:go_default_library",
-        "@com_github_wercker_journalhook//:go_default_library",
         "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
     ],
 )
@@ -67,7 +66,6 @@ go_image(
         "@com_github_sirupsen_logrus//:go_default_library",
         "@com_github_urfave_cli_v2//:go_default_library",
         "@com_github_urfave_cli_v2//altsrc:go_default_library",
-        "@com_github_wercker_journalhook//:go_default_library",
         "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
     ],
 )

--- a/slasher/main.go
+++ b/slasher/main.go
@@ -110,8 +110,6 @@ func main() {
 			logrus.SetFormatter(joonix.NewFormatter())
 		case "json":
 			logrus.SetFormatter(&logrus.JSONFormatter{})
-		case "journald":
-			journalhook.Enable()
 		default:
 			return fmt.Errorf("unknown log format %s", format)
 		}

--- a/slasher/main.go
+++ b/slasher/main.go
@@ -19,7 +19,6 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
 	"github.com/urfave/cli/v2/altsrc"
-	"github.com/wercker/journalhook"
 	prefixed "github.com/x-cray/logrus-prefixed-formatter"
 )
 

--- a/validator/BUILD.bazel
+++ b/validator/BUILD.bazel
@@ -31,7 +31,6 @@ go_library(
         "@com_github_sirupsen_logrus//:go_default_library",
         "@com_github_urfave_cli_v2//:go_default_library",
         "@com_github_urfave_cli_v2//altsrc:go_default_library",
-        "@com_github_wercker_journalhook//:go_default_library",
         "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
         "@org_golang_google_grpc//:go_default_library",
     ],
@@ -61,7 +60,6 @@ go_image(
         "//shared/debug:go_default_library",
         "//shared/featureconfig:go_default_library",
         "//shared/logutil:go_default_library",
-        "//shared/maxprocs:go_default_library",
         "//shared/params:go_default_library",
         "//shared/version:go_default_library",
         "//validator/accounts/v1:go_default_library",
@@ -74,9 +72,9 @@ go_image(
         "@com_github_sirupsen_logrus//:go_default_library",
         "@com_github_urfave_cli_v2//:go_default_library",
         "@com_github_urfave_cli_v2//altsrc:go_default_library",
-        "@com_github_wercker_journalhook//:go_default_library",
         "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
         "@org_golang_google_grpc//:go_default_library",
+        "//shared/maxprocs:go_default_library",
     ],
 )
 

--- a/validator/main.go
+++ b/validator/main.go
@@ -323,8 +323,6 @@ contract in order to activate the validator client`,
 			logrus.SetFormatter(f)
 		case "json":
 			logrus.SetFormatter(&logrus.JSONFormatter{})
-		case "journald":
-			journalhook.Enable()
 		default:
 			return fmt.Errorf("unknown log format %s", format)
 		}

--- a/validator/main.go
+++ b/validator/main.go
@@ -29,7 +29,6 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
 	"github.com/urfave/cli/v2/altsrc"
-	"github.com/wercker/journalhook"
 	prefixed "github.com/x-cray/logrus-prefixed-formatter"
 	"google.golang.org/grpc"
 )

